lib/python-parser/metadata-parser.ts

```diff
@@ -0,0 +1,47 @@
+import { specifierValidRange } from "./common";
+import { PythonPackage, PythonRequirement } from "./types";
+
+const PACKAGE_NAME = "Name: ";
+const PACKAGE_VERSION = "Version: ";
+const PACKAGE_DEPS = "Requires-Dist: ";
+const DEP_PARSE_REGEX = /^(?<name>[\w.-]+)(\s?\(?(?<specifier><|<=|!=|==|>=|>|~=|===)(?<version>[\w.]+)\)?)?/;
+export function getPackageInfo(fileContent: string): PythonPackage {
+  const lines = fileContent.split("\n");
+  let name = "";
+  let version = "";
+  const dependencies: PythonRequirement[] = [];
+  for (let line of lines) {
+    line = line.trim();
+    if (line.length === 0) {
+      continue;
+    }
+    if (line.startsWith(PACKAGE_NAME)) {
+      name = line.substring(PACKAGE_NAME.length);
+    } else if (line.startsWith(PACKAGE_VERSION)) {
+      version = line.substring(PACKAGE_VERSION.length);
+    } else if (line.startsWith(PACKAGE_DEPS)) {
+      const pythonPackage = parseDependency(
+        line.substring(PACKAGE_DEPS.length),
+      );
+      if (pythonPackage) {
+        dependencies.push(pythonPackage);
+      }
+    }
+  }
+  return { name: name.toLowerCase(), version, dependencies } as PythonPackage;
+}
+
+function parseDependency(dep: string): PythonRequirement | null {
```

Comment:
Is this the dependency name & version? Can we give the arg a more explanatory name?